### DISCUSSION TITLE
[Enhancement] [upload] upload组件增加上传文件夹功能

### DISCUSSION THREAD

**Discussion by dzdzdzd123** [element-plus/element-plus#17932] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `upload`

## Description

upload组件增加上传文件夹功能

### Additional comments

_(empty)_

**Comment by ZeMinCai** [comment]
Don't have that feature?

**Reply by dzdzdzd123** [reply]
This can only upload multiple files and does not have the function of uploading folders
